content/docs/command-reference/checkout.md

```diff
@@ -18,9 +18,10 @@ positional arguments:
 ## Description
 
 This command is usually needed after `git checkout`, `git clone`, or any other
-operation that changes the current [DVC files](/doc/user-guide/dvc-files). It
-restores the corresponding versions of the DVC-tracked files and directories
-from the <abbr>cache</abbr> to the workspace.
+operation that changes the current
+[DVC files](/doc/user-guide/project-structure). It restores the corresponding
```

Comment:
@shcheklein unfortunately this change, while improving the structure, kind of brings us back to square 1 regarding terminology (#1663). Now there's no "DVC Files" section so it may be unclear what we mean by "DVC Files" not that it links to https://dvc.org/doc/user-guide/project-structure (which now includes .dvcignore and the .dvc/ dir)...

Should we go with "metafiles" after all? Expand to "dvc.yaml/lock and .dvc files" (or the specific ones that apply in each mention, ad hoc)?

Reply:
Cc @dberenbaum

Reply:
We can do a tooltip (if we don't have one), I don't see a big problem that we don't have the DVC Files section now to be honest. Or Im missing something here.

We should not link to the Projects structure - that's I agree though.

Reply:
> We can do a tooltip

Yes I was also thinking a tooltip or addressing this as part of the Concepts section (#550).

Reply:
> We should not link to the Projects structure

So should I try to address that in this PR @shcheklein? Or put it in another new/existing issue for now?

Reply:
Any tooltip is probably better than a link to the Project Structure. It's up to you though!